compiler/rustc_transmute/src/layout/tree.rs

```diff
@@ -484,10 +484,6 @@ pub(crate) mod rustc {
         }
     }
 
-    pub(crate) fn ty_layout<'tcx>(cx: LayoutCx<'tcx, TyCtxt<'tcx>>, ty: Ty<'tcx>) -> Layout<'tcx> {
```

Comment:
Thanks for removing this. I was gonna ask for it to be removed if you hadn't done it, but you read my mind 😆